### PR TITLE
Add Version class to __all__ export. Fix #410

### LINKED ISSUE #410: Module "semver" does not explicitly export attribute "Version"
Hi

I am running `mypy` the python static type checker in my project with the `--strict` flag and it is causing an error. 

```
Module "semver" does not explicitly export attribute "Version"
```

I have replicated the error in this repl environment online
https://replit.com/@soneji/semver-mypy

You can click "show code" to see the code but I have printed it also

I have followed the usage instructions in the documentation so I am unsure why this error is being caused

### PR DESCRIPTION
**Pull request by Soneji**

### CODE REVIEW

File: src/semver/__init__.py [hunk 1]
```diff
@@ -37,3 +37,5 @@
     __maintainer_email__,
     SEMVER_SPEC_VERSION,
 )
+
+__all__ = ["Version"]
```
Comment (tomschr) [inline src/semver/__init__.py:41]: I fear, this is incomplete. It's not only `Version` we want to expose, but also the other imported functions, variables, and classes.

Could you add the other names as well?

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks @Soneji for your contribution! :+1: I have a minor comment, see below.

To avoid formatting errors, call `tox -e check` to see if everything is okay. Sometimes `black` complains about some lines. :wink:

File: src/semver/__init__.py (hunk 1 above)
Comment (Soneji) [reply src/semver/__init__.py:41]: Including the deprecated ones?

Comment (tomschr) [reply src/semver/__init__.py]: > Including the deprecated ones?

Yes. Some still use them so we can't omit them.

**Comment by Soneji** [comment]
Looks like the checks have passed. Is this ok to merge?

**Comment by tomschr** [comment]
Many thanks for the contribution! :+1: 

> Looks like the checks have passed. Is this ok to merge?

Yes, I can merge it.

**Comment by tomschr** [comment]
> Amazing, thank you so much

Thanks for your contribution! :heart: 

> Are you able to release and publish a patch for this?

Yes, I'm actually planning to create a new release. Not sure if I have time today, but it will be available very soon. I'll let you know. :slightly_smiling_face:

**Comment by tomschr** [comment]
@Soneji I was able to create it sooner than I thought. :wink: Find all information here (including PyPI):

https://github.com/python-semver/python-semver/releases/tag/3.0.1

**Pull request merged**